packages/hardware-trezor/src/transformers/key-paths.ts

```diff
@@ -0,0 +1,31 @@
+import { BIP32Path } from '@cardano-sdk/crypto';
+import { Cardano } from '@cardano-sdk/core';
+import { CardanoKeyConst, GroupedAddress, util } from '@cardano-sdk/key-management';
+import { TrezorTxTransformerContext } from '../types';
+
+/**
+ * Constructs the hardened derivation path of the payment key for the
+ * given grouped address of an HD wallet as specified in CIP 1852
+ * https://cips.cardano.org/cips/cip1852/
+ */
+export const paymentKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32Path => [
+  util.harden(CardanoKeyConst.PURPOSE),
+  util.harden(CardanoKeyConst.COIN_TYPE),
+  util.harden(address.accountIndex),
+  address.type,
+  address.index
+];
```

Comment:
This is a shared util / transformer used for both, Ledger and Trezor, data mappings. I know this is an isolated package but maybe we can consider that and share those utils by using e.g. key management utils?

Reply:
@tomislavhoracek yeah that's one of the things i wanted to discuss - i also thought about moving it to key-management (and actually had a TODO added but couldn't commit that since we cannot submit TODOs in SDK :D)

BUT this would be a change to the `hardware-ledger` package too, so i was hesitant and maybe we should do that refactoring / sharing utilities at the end? Same goes for the test data etc.